docs/book/src/getting-started/installation.md

```diff
@@ -72,11 +79,11 @@ secretproviderclasses.secrets-store.csi.x-k8s.io
 secretproviderclasspodstatuses.secrets-store.csi.x-k8s.io
 ```
 
-## Use the Secrets Store CSI Driver with a Provider
+## Install External Secret Providers
 
 Now that the Secrets Store CSI Driver has been deployed, select a provider from the supported provider list, then follow the installation steps for the provider:
 
+- [AWS Provider](https://github.com/aws/secrets-store-csi-driver-provider-aws)
```

Comment:
Do you want to update the order and links here too?

https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/docs/book/src/introduction.md
https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/docs/book/src/concepts.md

Reply:
sgtm